app/views/layouts/submissions/editor/_problem_form.html.erb

```diff
@@ -30,6 +30,8 @@
                        value: @default_content,
                        style: 'display: none' %>
         </div>
+        <%= f.hidden_field :editor_bottom, value: editor_bottom if defined?(editor_bottom) %>
```

Comment:
This means that this value, if present, will be always `true`. This is weird. I'd change this expression to 

```ruby
.... value: defined?(editor_bottom) ? editor_bottom : false
```

And then check that the value is true, instead than checking presence of the hidden field